src/views/Logout.vue

```diff
@@ -13,15 +13,11 @@ export default class Logout extends Vue {
 
     public async created() {
         const wallet = await WalletStore.Instance.get(this.request.walletId);
-        if (!wallet) {
-            this.$rpc.reject(new Error('Account ID not found'));
-            return;
-        }
 
         const request: KeyguardClient.RemoveKeyRequest = {
             appName: this.request.appName,
             keyId: this.request.walletId,
-            keyLabel: wallet.label,
+            keyLabel: !!wallet ? wallet.label : 'I want to log out', // the label to type in Keyguard to confirm logout
         };
```

Comment:
Double `!!` required?

Reply:
Not required.
Maybe nicer?
Shows that it's used as a boolean condition here.
But could equally be removed.

Reply:
What about I18n of that label? Wouldn't it be better to let keyguard handle that?

Reply:
i18n could also be handled here in AM once we add i18n to the AM

Reply:
Yeah ok, still it seems strange to me to misuse the request parameter like that.